sync/sync_head.go

```diff
@@ -191,6 +191,9 @@ func isExpired(header header.Header, period time.Duration) bool {
 
 // isRecent checks if header is recent against the given blockTime.
 func isRecent(header header.Header, recencyThreshold time.Duration) bool {
+	if recencyThreshold == 0 {
+		recencyThreshold = time.Second * 20 // this default is based on a block time of 15 seconds
```

Comment:
Why not make it based on blockTime, as discussed?

Reply:
ok i'll just allow the default to be the estimation we did before.

Reply:
check now pls